SKYKitChat/Classes/Core/Records/SKYConversation.m

```diff
@@ -54,6 +54,32 @@ + (instancetype)recordWithRecord:(SKYRecord *)record
     return conversation;
 }
 
++ (NSArray<NSString *> *)restrictedFieldKeys
+{
+    static NSArray<NSString *> *_restrictedFieldKeys;
+    if (!_restrictedFieldKeys) {
+        // this list is retrieved from:
+        //   https://github.com/SkygearIO/chat/blob/1.6.0-3/chat/conversation_handlers.py#L211
```

Comment:
From the plugin code, the record before_save will remove those attributes. So we should not need to do it in the client?

Reply:
SDK will include serialized message record in `last_message` field and skygear server will expect a record reference. The tricky point is that the schema conflict error occurs before calling `before_save`. 😅

Reply:
Oh, than I think we should also check the other SDK if it has similar issue? Let me check and merge this first.